components/config/caching.rst

```diff
@@ -1,26 +1,27 @@
 .. index::
    single: Config; Caching based on resources
 
-Caching Based on Resources
+Caching based on Resources
```

Comment:
I really get the feeling that we should think again about our capitalisation rule. :)